addons/calendar/models/calendar.py

```diff
@@ -927,7 +927,7 @@ def ics_datetime(idate, allday=False):
             # FIXME: why isn't this in CalDAV?
             import vobject
         except ImportError:
-            _logger.warning("The `vobject` Python module is not installed, so iCal file generation is unavailable. Use 'pip install vobject' to install it")
+            _logger.warning("The `vobject` Python module is not installed, so iCal file generation is unavailable. Use 'pip3 install vobject' to install it")
```

Comment:
Actually I don't like these warnings. The way we install dependency is system specific. We should modify into something like "install vobject python library".